spec/models/user_spec.rb

```diff
@@ -70,6 +78,7 @@
         expect {
           described_class.from_social_provider('provider', params)
         }.to change(described_class, :count).by(1)
+        expect(Flipper).not_to be_enabled(:test_feature)
```

Comment:
I'll remove this change before merging the PR, this is just a small POC within the PR

Reply:
This demonstrates that the feature flag does not save the state between different tests